src/System.Time/src/System/DateTimeEx.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System
+{
+    /// <summary>
+    /// Extension methods for <see cref="DateTime"/>.
+    /// </summary>
+    public static class DateTimeEx
```

Comment:
What should I call a class then that provides extension methods to the types not in this assembly?  Is `DateTimeExtensions` acceptable?